City-of-Helsinki/drupal-helfi-sote
# Add permissions to create and edit neighbourhoods taxonomy terms

How to test:
* Checkout this branch using existing site.
* Import changes using e.g. `make fresh`.
* Admin users should be able to create, edit, translate and delete terms in neighbourhoods taxonomy.
* Content editors users should be able to create, edit, translate and delete terms in neighbourhoods taxonomy.
